### PR TITLE
Don't expose LDAP authentication usage.

### PR DESCRIPTION
**Pull request by HLeithner**
As discussed in JSST, change translation strings in public tracker. We don't want to expose the usage of LDAP for authentications.

### Summary of Changes
Language file Updates

### CODE REVIEW

**Comment by brianteeman** [comment]
isnt that security through obscurity?

**Comment by mbabker** [comment]
It is.  But this is one of those cases where the public facing error message probably shouldn't say what authentication source is in use (same logic is already in use with Gmail auth from a cursory glance).

**Comment by brianteeman** [comment]
No the gmail plugin will error with
https://github.com/joomla/joomla-cms/blob/c4ff5155d2bf6cdfa506061c203e30400b8bb327/plugins/authentication/gmail/gmail.php#L197

where PLG_GMAIL_ERROR_LOCAL_USERNAME_CONFLICT="A local username conflicts with your Gmail username."

**Comment by mbabker** [comment]
That one's a very specific case.  But the other fail messages are the generic messages.  Unless I misread and the keys actually are referencing strings saying "could not connect to Google to authenticate" basically.

**Comment by infograf768** [comment]
these keys are specific to ladp afaik
I suggest, if you do not want in future releases to still show the word `ladp` when a language pack is not up to date, instead of modifying the value, adding new strings where NO is replaced by NOT (for example)
`JGLOBAL_AUTH_NOT_BIND`
Precede the old strings by a comment
`; The following 2 strings are deprecated and will be removed with 4.0.`

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 431e5e512aef3a9e13b820447ca23da6cf6b8fe1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18531">issues.joomla.org/tracker/joomla-cms/18531</a>.</sub>

**Comment by HLeithner** [comment]
I followed @infograf768 suggestion and added 2 new strings.

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on 69623e6658976a025d1a3ecf30592bd2f66c6ebc<br /><br />Thanks @HLeithner <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18531">issues.joomla.org/tracker/joomla-cms/18531</a>.</sub>

**Comment by zero-24** [comment]
@brianteeman can you please give your ok as code owner for the language files so we can move that one forward into 3.8.8? Thanks.

**Comment by brianteeman** [comment]
No need to ask me. GitHub tells me but I am in the pub so will check later,

**Review by brianteeman — CHANGES_REQUESTED** [review]
No point in adding two new strings that are identical

**Comment by zero-24** [comment]
Fixed please review again @HLeithner @brianteeman ;)

File: administrator/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -293,8 +293,10 @@ JGLOBAL_AUTH_FAILED="Failed to authenticate: %s"
 JGLOBAL_AUTH_INCORRECT="Incorrect username/password"
 JGLOBAL_AUTH_INVALID_PASS="Username and password do not match or you do not have an account yet."
 JGLOBAL_AUTH_INVALID_SECRETKEY="The two factor authentication Secret Key is invalid."
+; The following 2 strings are deprecated and will be removed with 4.0.
 JGLOBAL_AUTH_NO_BIND="Unable to bind to LDAP"
 JGLOBAL_AUTH_NO_CONNECT="Unable to connect to LDAP server"
+JGLOBAL_AUTH_NOT_CONNECT="Unable to connect to authentication service."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.ini]: Sort in alpha order.

**Comment by zero-24** [comment]
done @Quy thanks.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c7c0c4074e14cc97482cfeb8b5dcfcfc76d2d8be<br /><br />Code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18531">issues.joomla.org/tracker/joomla-cms/18531</a>.</sub>

**Review by brianteeman — APPROVED** [review]

**Comment by zero-24** [comment]
RTC. Thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18531">issues.joomla.org/tracker/joomla-cms/18531</a>.</sub>

**Comment by HLeithner** [comment]
There is a reason for 2 strings but only for debugging usage so I'm fine with only one string

**Pull request merged**